packages/documentation-site/patternfly-docs/content/design-guidelines/components/pagination/pagination.md

```diff
@@ -6,65 +6,86 @@ section: components
 
-#### Desktop full pagination in toolbar
+### Compact pagination
 
 <div class="ws-docs-content-img">
-![an example of a desktop pagination component within a toolbar.](./img/toolbar-pagination.svg)
+![Example of a compact pagination.](./img/compact-pagination.svg)
 </div>
 
-1. **Items in view:** Allows the user to select the item count (number of listed items) per page.
+1. **Per-page selector:** Used to choose the number of items displayed on a single page at once.
+2. **Previous page/next page:** Used to navigate back or forward one page of content at a time.
+
+Compact pagination reduces visual crowding and saves space for additional actions in limited spaces, like toolbars. 
 
-2. **Full pager:** Supplies all necessary pagination options including page-back and page-forward, page-first and page-last, and a manual option to type in a desired page number.
+### Indeterminate pagination
 
-#### Mobile full pagination in toolbar
+When it's difficult to calculate the total number of items or page count upfront, you can use an indeterminate pagination, which replaces the total page number in the per-page selector with a general label of “many”. Users can still choose the number of items shown at once, and the indeterminate label can be used within both full and compact pagination components.
 
 <div class="ws-docs-content-img">
-![Example of pagination on mobile table view.](./img/mobile-pagination.svg)
+![Example of a indeterminate pagination.](./img/indeterminate-pagination.svg)
 </div>
 
-1. **Items in view:** At this screen size, the top toolbar only shows the item count to indicate items currently in view. It hides all other pagination controls.
+## Placement
 
-2. **Full pager:** Supplies the same pagination options as the desktop full pager, but in the bottom toolbar.
+### Desktop
 
-### Compact pagination
-
-Compact pagination fits more actions into top toolbars with limited space. Choose this compact variation when your top toolbar contains many items or looks overcrowded. As with full pagination, the compact variation condenses to an item count for mobile views.
-
-#### Desktop compact pagination in toolbar
+For desktop views, place a full pagination component above the content it controls,  typically within a toolbar.
 
 <div class="ws-docs-content-img">
-![Example of a compact pagination.](./img/compact-pagination.svg)
+![an example of a desktop pagination component within a toolbar.](./img/toolbar-pagination.svg)
 </div>
 
-1. **Items in view:** Allows the user to select the item count (number of listed items) per page, as seen in full pagination.
+1. **Per-page selector:**  Used to choose the number of items displayed on a single page at once.
 
-2. **Compact pager:** Supplies the user with page-back and page-next controls only.
+2. **Full pagination:** Supplies data navigation controls, including previous page and next page, first page and last page, and a manual option to type in a desired page number.
```

Comment:
These bullets and the purple numbers in the image don't really make sense, but it could just be me 😅 "Per page selector" is describing the 1 box in the image, but that's already covered in the full pagination docs above - do we need to include it again? The 2 box in the image points to the "next page" button, but it says "Full pagination" here with a general description of full pagination. I might just remove these 2 bullets - IMO they aren't needed. You could also link "full pagination" in the sentence above this image to the full pagination heading of this doc, which would point people to the full pagination docs covered in these bullets.